Mathlib/Topology/OmegaCompletePartialOrder.lean

```diff
@@ -116,11 +120,7 @@ theorem scottContinuous_of_continuous {α β} [OmegaCompletePartialOrder α]
 
 theorem continuous_of_scottContinuous {α β} [OmegaCompletePartialOrder α]
     [OmegaCompletePartialOrder β] (f : Scott α → Scott β)
-    (hf : OmegaCompletePartialOrder.Continuous' f) : Continuous f := by
+    (hf : ωScottContinuous f) : _root_.Continuous f := by
```

Comment:
Why is the new `_root_` needed? It looks like the namespaces haven't changed here

Reply:
Both `OmegaCompletePartialOrder.Continuous` and `Continuous` were in sight here (with opened namespace). I think 
 it was just a lucky dip that it worked before without the fully qualified name.